src/mmw/sass/pages/_model.scss

```diff
@@ -61,4 +61,8 @@
       }
     }
   }
+
+  .model-results-tooltip {
+    color: #333333;
```

Comment:
The icon get's a little cutoff
<img width="52" alt="screen shot 2017-07-13 at 8 24 14 am" src="https://user-images.githubusercontent.com/7633670/28166188-feeaabde-67a4-11e7-95f6-569c4450adb7.png">

Consider adding a width to it
```css
i {
    width: 15px;
}
```